pkg/log/log_messages.go

```diff
@@ -75,3 +75,5 @@ const CAKC059 string = "CAKC059 Path exists but does not contain regular file: %
 const CAKC060 string = "CAKC060 Setting %s given invalid value %s"
 const CAKC061 string = "CAKC061 Failed to validate setting for Authenticator configuration"
 const CAKC062 string = "CAKC062 Required Authenticator setting %s not provided"
+const CAKC063 string = "CAKC063 Can't find configuration for URL : %s"
+const CAKC064 string = "CAKC064 Can't find Authenticator for this configuration"
```

Comment:
@tzheleznyak  - What is the use-case for this?

Reply:
We are going to add support to authn-jwt to authn-k8s-client.
And this is the error message when the authn-client can't decide which authentication flow should run (authn-k8s or authn-jwt)

Reply:
How about "No authenticator configured. "

Reply:
or even better: "No authentication configured. "